search-app/docs/api.md

```diff
@@ -1686,33 +1686,46 @@ Find collections matching service concept id.
 ##### <a name="c-polygon"></a> Polygon
 
 Polygon points are provided in counter-clockwise order. The last point should match the first point to close the polygon. The values are listed comma separated in longitude latitude order, i.e. lon1, lat1, lon2, lat2, lon3, lat3, and so on.
+This parameter supports the and/or option as shown below.
```

Comment:
This is not what I had in mind when I filed CMR-6434. From the description of CMR-6434:
```
For now, we will just add a single option `options[spatial][or]=true` to apply the OR option across all spatial shapes. The current idea is to only apply this option on explicit shapes, such as point, bounding_box, polygon, circle, etc, but not to shape file or geojsons.
```
The idea is that any spatial condition within search will be ORed together regardless of the specific spatial types. I have thought that would be more useful and the behavior more consistent. 

with this implementation, only specific spatial types can be ORed, but between spatial types, it is still ANDed together and can't be changed.